thunder/torch/__init__.py

```diff
@@ -4936,6 +4936,14 @@ def _set_grad_enabled_with_warning(enabled: bool) -> None:
 register_function(torch._C._set_grad_enabled, _set_grad_enabled_with_warning)
 
 
+def _unwrap_if_dead(tensor):
+    warnings.warn("torch._C._functorch.unwrap_if_dead have no effect under thunder.jit")
```

Comment:
```suggestion
    warnings.warn("torch._C._functorch.unwrap_if_dead has no effects under thunder.jit.")
    warnings.warn("`torch.autograd.Function.backward is not respected by `thunder.jit`. `thunder.jit` generates backward based on the forward")
```

Reply:
can we use warn once? Mentioning autograd.Function reasonable to me to do. Maybe we should also file an issue to track / design autograd.Function support, I'll do that.